packages/rps/package.json

```diff
@@ -34,7 +36,7 @@
     "precommit": "lint-staged --quiet"
   },
   "lint-staged": {
-    "{src,tests}/**/*.{ts,tsx}": "yarn lint:check"
+    "{src,tests}/**/*.{ts,tsx}": "eslint"
```

Comment:
Since you're using `.eslintignore` and `**/*.{js,ts,tsx}` for the linting, maybe you should use that regex here too.

Reply:
So I should change this back to `yarn lint:check`?

Reply:
So it checks Prettier as well?

Reply:
Oh nevermind you meant on the left side :)